includes/cub.h

```diff
@@ -12,7 +12,7 @@
 # include "mlx.h"
 # include "libft.h"
 
-# define WIDTH 960//1920
+# define WIDTH 1220//1920
 # define HEIGHT 720//1080
```

Comment:
What's up with changing this all the time? 

I propose we use common monitor resolutions such as 800x600, 1280x720 or 1600x900?

Reply:
Ha I don't know, I like to "keep" the big dimensions cos they are the full screen on school imacs if I remember correctly, but then in mine I just tuned it to something that looked nicer, 960x720 was too squarish, but ya not really a very important thing. 1280x720 sounds good, we can keep those if you want :)